app/main.py

```diff
@@ -2,24 +2,60 @@
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_value: int, max_value: int) -> None:
+        self.min_value = min_value
+        self.max_value = max_value
+
+    def __set_name__(self, instance: object, name: str) -> None:
+        self.protected_name = f"_{name}"
+
+    def __get__(self, instance: object, owner: object) -> str:
+        return getattr(instance, self.protected_name)
+
+    def __set__(self, instance: object, value: int) -> None:
+        if not isinstance(value, int):
+            raise TypeError(f"{self.protected_name} must be integer!")
+        if not self.min_value <= value <= self.max_value:
+            raise ValueError(f"Value must be in range from "
+                             f"{self.min_value} to {self.max_value}")
+        setattr(instance, self.protected_name, value)
 
 
 class Visitor:
-   pass
+    def __init__(self, name: str, age: int, weight: int, height: int) -> None:
+        self.name = name
+        self.age = age
+        self.weight = weight
+        self.height = height
```

Comment:
This is a code quality issue. The `Visitor` class does not use any kind of type checking or validation for its attributes. This can lead to bugs and errors in the future. It would be better to use the `IntegerRange` class for the `age`, `weight`, and `height` attributes to ensure they are within the correct ranges.